### PR TITLE
Added new blog post

### PR DESCRIPTION
**Pull request by olenagerasimova**

### CODE REVIEW

File: _posts/2023/05/2023-05-02-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - Security policy
+  - Permissions
+  - Filtering
+  - Rest API
+---
+
+Artipie new version [v0.30.0](https://github.com/artipie/artipie/releases/tag/v0.30.0) was released on May 2. 
+In the previous month we added several features, here are the main three of them:
+
+- Artifacts filtering provides means to filter out resources of a repository by specifying patterns of resource location.
```
Comment (dgarus) [inline _posts/2023/05/2023-05-02-release-notes.md:23]:
```suggestion
- Artifacts filtering provides a means to filter out resources of a repository by specifying patterns of resource location. 
```

File: _posts/2023/05/2023-05-02-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Permissions for REST API operation were added. Now, it's possible to configure access for API endpoints and control
+what users are allowed to view and/or manage repositories, storage aliases, users, roles. All the details are described 
+in [wiki](https://github.com/artipie/artipie/wiki/Configuration-Policy#rest-api-permissions).
+
+- [S3 storage configuration](https://github.com/artipie/artipie/wiki/Configuration-Storage#s3-storage) does not require 
+credentials in YAML configuration anymore. Credentials for S3 storage can set via Java System Properties, environmental 
+variables, in `~/.aws/credentials` file or any other way supported by default AWS provider.
+
+Thanks for reading, if you have any question or suggestion feel free to [create an issue](https://github.com/artipie/artipie/issues/new) or contact us in [Telegram](https://t.me/artipie).
```
Comment (dgarus) [inline _posts/2023/05/2023-05-02-release-notes.md]:
```suggestion
Thanks for reading, if you have any questions or suggestions, feel free to [create an issue](https://github.com/artipie/artipie/issues/new) or contact us on [Telegram](https://t.me/artipie).
```

**Review by dgarus — CHANGES_REQUESTED** [review]
thanks!
there are minor comments.

File: _posts/2023/05/2023-05-02-release-notes.md (hunk 1 above)
Comment (olenagerasimova) [reply _posts/2023/05/2023-05-02-release-notes.md:23]: that's defiantly not correct as word "means" is plural and supposition "a" can be used only with singular

**Comment by olenagerasimova** [comment]
@dgarus thanks, corrected and answered

**Review by dgarus — APPROVED** [review]

**Pull request merged**